config/storybook-preset-carbon/index.js

```diff
@@ -26,12 +26,12 @@ module.exports = {
     '@storybook/addon-storysource',
     '@storybook/addon-knobs',
     '@storybook/addon-actions',
-    '@storybook/addon-links',
     '@storybook/addon-docs',
     '@storybook/addon-notes/register',
```

Comment:
Getting a few warnings regarding `/register` 🤔 
<img width="447" alt="Screen Shot 2021-01-21 at 3 47 04 PM" src="https://user-images.githubusercontent.com/11928039/105426400-eb747c80-5bff-11eb-8719-a5f23664c48a.png">

Reply:
its coming from the preset.js I'm unsure of what it wants since it seems to be working and those docs no longer work.